packages/sdg/src/lib/alert/alert.component.ts

```diff
@@ -40,6 +42,22 @@ export class AlertComponent {
   iconName = computed(() => AlertIcon[this.type()]);
   typeName = computed(() => AlertType[this.type()]);
 
+  private messageValidation(message: string): string {
+    if (this.isCloseable()) {
+      if (message.length > 105) {
+        return `${message.slice(0, 105)}...`;
+      }
+
+      return message;
+    } else {
+      if (message.length > 120) {
+        return `${message.slice(0, 120)}...`;
+      }
+
+      return message;
+    }
```

Comment:
On pourrait améliorer la lisbilité en utilisant les "early return" pour éviter les conditions imbriqués
```suggestion
    if (this.isCloseable() && message.length > 105) {
      return `${message.slice(0, 105)}...`;
    }

    if (message.length > 120) {
      return `${message.slice(0, 120)}...`;
    }

    return message;
```